dd-sdk-android/src/main/kotlin/com/datadog/android/Datadog.kt

```diff
@@ -123,7 +125,9 @@ object Datadog {
         }
         val okHttpClient = OkHttpClient.Builder()
             .addInterceptor(networkTimeInterceptor)
+            .addInterceptor(gzipInterceptor)
             .callTimeout(NETWORK_TIMEOUT_MS, TimeUnit.MILLISECONDS)
+            .writeTimeout(NETWORK_TIMEOUT_MS, TimeUnit.MILLISECONDS)
```

Comment:
if it takes too long time to write (encode) it will drop the package ?

Reply:
No it won't, it will retry later. 
The idea with this change is that by default the write timeout is set to 10 seconds. Since the biggest part of the calls we perform is writing (sending logs), I increased it to match the call timeout (call = connect + write + read). 
FYI default connect timeout is 0 (no timeout) and read is 10s.